src/Take.Blip.Builder/Actions/ProcessHttp/ProcessHttpAction.cs

```diff
@@ -73,6 +78,15 @@ public ProcessHttpAction(IHttpClient httpClient, ILogger logger)
             }
         }
 
+        private async Task AddUserToHeaderAsync(HttpRequestMessage httpRequestMessage, IContext context, CancellationToken cancellationToken)
+        {
+            var userHeaderValue = await context.GetVariableAsync("config.processHttp.addUserToRequestHeader", cancellationToken);
```

Comment:
Instead of accessing the config like that, you can use `context.Flow.Configuration` that directly provide the configured keys without the overhead of parsing the variable provider

Reply:
So it should be like this (I will do the proper key existence test), removing the 'config' prefix?

var userHeaderValue = context.Flow.Configuration["processHttp.addUserToRequestHeader"];

BTW, the '.' could cause an error since it work as a 'navigation' indicator?

Reply:
Yes, the prefix is used internally to determine which variable provider should be used. The config variable provider uses the configuration dictionary in the flow model. The '.' for navigation is just a convention and should not be an issue.